gatsby-config.js

```diff
@@ -1,13 +1,13 @@
 module.exports = {
   plugins: [
     // Comment and uncomment the desired CMS/backend plugin to switch between data sources
-    "contentful-plugin",
-    // "datocms-plugin",
+    // "contentful-plugin",
+    "datocms-plugin",
     // "drupal-plugin",
     // "wordpress-plugin",
 
     // optional blog themes
-    // "gatsby-theme-contentful-blog",
+    "gatsby-theme-contentful-blog",
     // "gatsby-theme-datocms-blog",
```

Comment:
Also curious how you're thinking we'll handle the blog theme that we keep in tact for each published starter dist — just pick one and stick with it?

Reply:
Makes sense!